jsonnetsecure/jsonnet.go

```diff
@@ -79,12 +70,6 @@ func WithProcessIsolatedVM(ctx context.Context) Option {
 	}
 }
 
-func WithExecTimeout(timeout time.Duration) Option {
```

Comment:
If you remove this it needs an update in depending services as well!

Reply:
I grepped our code bases and found no uses. :)